_get_started/mobile.md

```diff
@@ -10,11 +10,29 @@ published: true
 
 ## Get Started with PyTorch ExecuTorch
 
-<p>
-    <a href="https://pytorch.org/executorch/stable/index.html" class="btn btn-lg with-right-arrow">
-    ExecuTorch Documentation
-    </a>
-</p>
+PyTorch’s edge specific library is [ExecuTorch(https://github.com/pytorch/executorch/)] and is designed to be lightweight, very performant even on devices with constrained hardware such as mobile phones, embedded systems and microcontrollers.
```

Comment:
looks like the link is broken in the preview: https://deploy-preview-1935--pytorch-dot-org-preview.netlify.app/get-started/executorch/ 
```suggestion
PyTorch’s edge specific library is [ExecuTorch](https://github.com/pytorch/executorch/)] and is designed to be lightweight, very performant even on devices with constrained hardware such as mobile phones, embedded systems and microcontrollers.
```